stat.go

```diff
@@ -934,18 +929,19 @@ func MeanVariance(x, weights []float64) (mean, variance float64) {
 			compensation += d
 		}
 		variance = (ss - compensation*compensation/float64(len(x))) / float64(len(x)-1)
-		return
```

Comment:
The syntactic pattern `if cond { // code; return }; // other code` is standard in Go.